.github/workflows/get-secretmanager-secrets-it.yml

```diff
@@ -0,0 +1,47 @@
+name: get-secretmanager-secrets Integration
+on: [push]
+jobs:
+  gcloud:
+    name: with setup-gcloud
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./setup-gcloud
+      with:
+        service_account_email: ${{ secrets.SETUP_GCLOUD_IT_EMAIL }}
+        service_account_key: ${{ secrets.SETUP_GCLOUD_IT_KEY_B64 }}
+    - id: secrets
+      uses: ./get-secretmanager-secrets
+      with:
+        secrets: |-
+          token:foo/bar
+    - name: get output
+      run: echo '${{steps.secrets.outputs.token}}'
+
+  b64_json:
+    name: with base64 json creds
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - id: secrets
+      uses: ./get-secretmanager-secrets
+      with:
+        credentials: ${{ secrets.SETUP_GCLOUD_IT_KEY_B64 }}
+        secrets: |-
+          token:${{ secrets.GET_SECRETMANAGER_SECRET_REF }}
```

Comment:
Need to create this GitHub secret. That will involve also creating a secret in Secret Manager. Not sure if you have a GCP project that you're already using for tests.

Reply:
We do, will send info for it offline.